sky/packages/sky/lib/widgets/scrollable.dart

```diff
@@ -137,27 +127,31 @@ abstract class Scrollable extends StatefulComponent {
     super.didUnmount();
   }
 
-  bool scrollTo(double newScrollOffset, { ValueAnimation<double> animation }) {
+  Future scrollTo(double newScrollOffset, { Duration duration, Curve curve: ease }) {
     if (newScrollOffset == _scrollOffset)
-      return false;
+      return new Future.value();
 
-    if (animation == null) {
+    Future returnValue;
+    if (duration == null) {
+      Completer completer = new Completer();
+      returnValue = completer.future;
       setState(() {
         _scrollOffset = newScrollOffset;
+        completer.complete();
```

Comment:
The setState function calls this function synchronously, so there's no need for a Completer.  You can just return `new Future.value()` as you do above.